src/gui/guiFormSpecMenu.cpp

```diff
@@ -1326,6 +1327,50 @@ void GUIFormSpecMenu::parseField(parserData* data, const std::string &element,
 	errorstream<< "Invalid field element(" << parts.size() << "): '" << element << "'"  << std::endl;
 }
 
+void GUIFormSpecMenu::parseHyperText(parserData* data, const std::string &element)
+{
+	std::vector<std::string> parts = split(element,';');
+	if (parts.size() == 4)
+	{
```

Comment:
Misplaced bracket. Error & return would also reduce the indent level.
Consider adding a forwards-compatibility mode like the other formspec elements. In theory it is possible to let mods specify the `formspec_version[n]` element, so for future changes that would be handy to not require adding a new element.